x-pack/plugins/security_solution/public/risk_score/containers/all/index.tsx

```diff
@@ -5,21 +5,10 @@
  * 2.0.
```

Comment:
Just a note that it might be worth breaking these apart into separate files. (i.e. use_user_risk_score.tsx, use_host_risk_score.tsx and they both import from use_risk_score.tsx. Not necessary for this PR, but maybe something to do in the future if there's time

Reply:
mm, I guess they were bind together because they were very much alike, so didn't change that bit. Happy to have more discussion about it with @machadoum later.